bot.py

```
@@ -136,6 +141,11 @@ async def text_dice(callback_query: types.CallbackQuery, bot: Bot):
     await callback_query.answer(f"Выпало: {dice_result}", show_alert=True)
 
 
+@router.message(lambda msg: msg.text == 'Memory' or msg.text == 'Память')
+async def memory(message: Message):
+    await message.reply('Выберете действие с памятью', reply_markup=memory_keyboard)
```

Ruff (0.12.2)

String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)